src/schemes/fluid/transport_velocity.jl

```diff
@@ -0,0 +1,200 @@
+"""
+    TransportVelocityAdami(background_pressure::Real)
+
+Transport Velocity Formulation (TVF) to suppress pairing and tensile instability.
+See [TVF](@ref transport_velocity_formulation) for more details on the method.
+
+# Arguments
+- `background_pressure`: Background pressure. Suggested is a background pressure which is
+                         on the order of the reference pressure.
+
+!!! note
+    There is no need for an artificial viscosity to suppress tensile instability when using `TransportVelocityAdami`.
+    Thus, it is highly recommended to use [`ViscosityAdami`](@ref) as viscosity model,
+    since [`ArtificialViscosityMonaghan`](@ref) leads to bad results.
```

Comment:
Honestly I don't know so far. I can imaging that it has something to do with dissipation. TVF is to get rid of the artificial viscosity and it's artificial dissipation of momentum. With TVF, there is an correction term in the momentum equation which acts as an effective stress to suppress clumping or tensile and pairing instability.  Both, TVF and artificial viscosity together, might lead to bad results. I haven't found a proof for this yet.

Reply:
So this is a quote?

Reply:
No, I noticed it when simulating with artificial viscosity. It led to bad results.

Reply:
So it diverged or it clumped?

Reply:
this are the errors for the taylor green vortex 2d.
Blue curve is with artificial viscosity and red with the physical

![image](https://github.com/user-attachments/assets/e2591061-9163-4705-ae9f-46a3af2619ad)

Reply:
One possible explanation:
The physical behavior of the fluid is altered when simulated with artificial viscosity. That is, it becomes more viscous than it would be at the given Reynolds number.

As we can see in the image below, the vortex decay is slower with artificial viscosity.

![image](https://github.com/user-attachments/assets/8b99e4b9-33b4-47d0-9416-a1008f3df4a8)

Reply:
Did you try to tune the artificial viscosity? Or just use the same value as always?

Reply:
The question is anyway if we should use unphysical viscosity if we want to model something in any of our applications.

Reply:
I computed `alpha` according to our docs:

![image](https://github.com/user-attachments/assets/fde595a8-5b33-4bf0-b045-dc4411870ed8)

Reply:
> Or just use the same value as always?

do you mean `alpha = 0.02`?

Reply:
I mean, why should I start tuning `alpha` only to get better results when I just can use the physical viscosity?

Reply:
> The question is anyway if we should use unphysical viscosity if we want to model something in any of our applications.

I think for low `alpha` an artificial viscosity is usable. However, if we attach importance to different Reynold numbers, we should not use artificial viscosity.